precompiles/ArbOwnerPublic.go

```diff
@@ -11,14 +11,25 @@ import (
 // The calls to this precompile do not require the sender be a chain owner.
 // For those that are, see ArbOwner
 type ArbOwnerPublic struct {
-	Address addr // 0x6b
+	Address                    addr // 0x6b
+	ChainOwnerRectified        func(ctx, mech, addr) error
+	ChainOwnerRectifiedGasCost func(addr) (uint64, error)
 }
 
 // GetAllChainOwners retrieves the list of chain owners
 func (con ArbOwnerPublic) GetAllChainOwners(c ctx, evm mech) ([]common.Address, error) {
 	return c.State.ChainOwners().AllMembers(65536)
 }
 
+// RectifyChainOwner checks if the account is a chain owner
+func (con ArbOwnerPublic) RectifyChainOwner(c ctx, evm mech, addr addr) error {
```

Comment:
One other thing I forgot to mention, we need to make sure this method is only accessible on ArbOS 11 to make sure nodes agree on if this method exists at any point in time. Search for `GetInfraFeeAccount` in `precompiles/precompile.go` for an example of how to do this.